cpp/include/cudf/detail/utilities/device_atomics.cuh

```diff
@@ -142,18 +141,17 @@ struct genericAtomicOperationImpl<T, Op, 8> {
     static_assert(sizeof(T) == sizeof(T_int));
```

Comment:
On line 140, same comment as above -- use `uint64_t` instead of `unsigned long long int`.

Reply:
I refer to keep `unsigned int` and `unsigned long long int` for consistency: These types are used in many places in this file. We should change all or none.

Reply:
That's fine. Let's leave it as-is and revisit later.